sdk/python/kfp/v2/components/importer_node_test.py

```diff
@@ -1,164 +0,0 @@
-# Copyright 2020 The Kubeflow Authors
```

Comment:
nit: ideally we should still keep a unit test file for this. Maybe test calling dsl.importer() with some value and then assert we got the right pipeline task object?
Given that we have the some test coverage in the compiler cli test, we could delay this a bit in a later PR.

Reply:
Okay, will add some tests in following PRs.